src/AppKit/NSBezierPath.cs

```diff
@@ -88,6 +88,11 @@ public unsafe void SetAssociatedPointsAtIndex (CGPoint[] points, nint index)
 				_SetAssociatedPointsAtIndex ((IntPtr)ptr, index);
 		}
 
+		public void Append (CGPoint[] points)
+		{
+			AppendPathWithPoints (points);
```

Comment:
that should be the opposite, the new/better API should have the implementation
the older/worse API should have an `[Obsolete]` and be put under `#if !XAMCORE_4_0`